examples/Exploring-different-models.ipynb

```diff
@@ -0,0 +1,802 @@
+{
```

Comment:
<p>If you can indicate what method or function caused NVT to export the schema, please do.</p><p> </p><p>There might be some unnecessary repetition in the second sentence. Maybe:</p><p> </p><p>&lt;fiction-warning please-check="true"&gt;</p><p>"The schema object identifies the input features. The Merlin Models library uses the schema to automatically build the layers to represent, normalize, and aggregate the input features."</p><p>&lt;/fiction-warning&gt;</p><p> </p><p>Sugg: lower case schema and init cap Merlin Models.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/256/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>thanks.</p>